packages/web-pkg/src/composables/actions/useBlockedPopupRedirectMessage.ts

```diff
@@ -0,0 +1,14 @@
+import { useGettext } from 'vue3-gettext'
+
+export const useBlockedPopupRedirectMessage = () => {
+  const { $gettext } = useGettext()
+
+  return {
+    title: $gettext('Blocked pop-ups and redirects'),
+    timeout: 20,
+    status: 'warning',
+    desc: $gettext(
+      'Some features might not work correctly. Please enable pop-ups and redirects in Settings > Privacy & Security > Site Settings > Permissions'
```

Comment:
Please use a more generic message that applies to all browsers:
**"Please turn on pop-ups and redirects in your browser settings to make sure everything works right."**